docs-src/go/chapter-durable-execution/non-deterministic-code-changes.md

```diff
@@ -0,0 +1,70 @@
+---
+id: non-deterministic-code-changes
+title: Non-deterministic code changes
+description: History Replay, sometimes also called Workflow Replay, is the mechanism that Temporal uses to reconstruct the state of a Workflow Execution. Temporal provides Durable Execution via this Replay Functionality.
+sidebar_label: Durability through Replays
+tags:
+  - go sdk
+  - developer-guide-doc-type
+  - event history
+  - replay
+  - durable execution
+---
+
+The most important thing to take away from the section is to make sure you have an application versioning plan whenever you are developing and maintaining a Temporal Application that will eventually deploy to a production environment.
+
+We cover versioning APIs and versioning strategies in other parts of the developer's guide, this chapter set the stage to understand why and how to approach those strategies.
+
+<!--TODO ^ update with links to those places -->
+
+### The Event History
+
+Let's inspect the Event History of a recent Background Check Workflow using the `temporal workflow show` command:
+
+```shell
+temporal workflow show \
+ --workflow-id backgroundcheck_workflow \
+ --namespace backgroundcheck_namespace
+```
+
+You will see output similar to this:
```

Comment:
Switching voice.

Reply:
This is probably a result of pick and mixing some of the existing language from the 102 course — I tried to be on top of this, but a good thing to be aware of in the future for these.